client/src/components/cityview/realm/villagers/NpcPanel.tsx

```diff
@@ -85,9 +85,13 @@ export const NpcPanel = ({ type = "all" }: NpcPanelProps) => {
   }, [realmId]);
 
   const gatherVillagers = () => {
+    const npcsNoBigInt = npcs.map((npc): any => {
```

Comment:
We can prob find a better name for this var

Reply:
But I agree, I rushed that one

Reply:
Changed :)